recipe/meta.yaml

```diff
@@ -1,46 +1,61 @@
 {% set name = "gguf" %}
-{% set version = "0.6.0" %}
+{% set version = "0.10.0" %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gguf-{{ version }}.tar.gz
-  sha256: b2e22eaba2a106c1ad148186a14ad9f29c429351686fe9d7ce16df39a0607e65
+  sha256: 52a30ef26328b419ffc47d9269fc580c238edf1c8a19b5ea143c323e04a038c1
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
   entry_points:
-    - gguf-convert-endian = "scripts:gguf_convert_endian_entrypoint"
-    - gguf-dump = "scripts:gguf_dump_entrypoint"
-    - gguf-set-metadata = "scripts:gguf_set_metadata_entrypoint"
+    - gguf-convert-endian = scripts:gguf_convert_endian_entrypoint
+    - gguf-dump = scripts:gguf_dump_entrypoint
+    - gguf-set-metadata = scripts:gguf_set_metadata_entrypoint
+    - gguf-new-metadata = scripts:gguf_new_metadata_entrypoint
+  skip: True # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
+    - poetry-core >=1.0.0
     - pip
-    - poetry-core
   run:
-    - python >=3.8
+    - python
     - numpy >=1.17
+    - tqdm >=4.27
+    - pyyaml >=5.1
+    - sentencepiece >=0.1.96,<0.2.0 # This dependency is missing in the upstream pyproject.toml file: https://github.com/ggerganov/llama.cpp/blob/master/gguf-py/pyproject.toml
```

Comment:
The bounds do not match your proposed change in the PR, is that expected?
`sentencepiece >=0.1.96,<0.2.0`
vs
`sentencepiece = ">=0.2.0"`

Reply:
@cbouss I updated it because the bounds causes problems with the llama.cpp-tools work, the newer version of sentencepiece causes conflicts, should I change it?

Reply:
FYI, I reported the upstream https://github.com/ggerganov/llama.cpp/issues/9248

Reply:
Serhii, there was already a comment in the recipe with a link to a PR: ttps://github.com/ggerganov/llama.cpp/pull/9107

Reply:
@jnoller I see llama.cpp-tools has sentencepiece <0.2.0. I am not sure what sentencepiece bound should be set here, but that can be changed later if needed.

Reply:
Ah, sorry, I didn't see it.